ui/ops/src/routes/ops/security-events/route.js

```diff
@@ -0,0 +1,18 @@
+export default [
+  {
+    path: 'security-events',
+    components: {
+      default: () => import('./SecurityEventsTable.vue'),
+      sidebar: () => import('./SecurityEventsSideBar.vue')
```

Comment:
I don't think this route has a sidebar, I couldn't find a way to open it at least.